lib/collections/schemas/accounts.js

```diff
@@ -107,6 +107,11 @@ export const Accounts = new SimpleSchema({
     type: Profile,
     optional: true
   },
+  groups: {
+    type: [String],
+    optional: true,
+    defaultValue: []
```

Comment:
No. It's just means at the schema level, they don't belong to a group yet. Although, on account creation, the array then receives either "guest" or "customer".